shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -41,6 +49,18 @@
     @Inject
     BridgeExecutorService bridgeExecutorService;
 
+    @Inject
```

Comment:
I need to make this dependency optional to be able to unit test the shard without deploying a keycloak instance. Maybe we can discuss that